.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  xcode-build:
+    name: Xcode Build
+    runs-on: macOS-latest
```

Comment:
oooooh can we drop this version back to use something a little earlier than the latest?

Reply:
Unfortunately they don't have anything older that 10.15. 😞  Our options are 10.15 (latest released) or 11.0 (in beta).